src/nav/Navigation.js

```diff
@@ -1,7 +1,9 @@
+/* @flow */
 /* eslint-disable react-native/split-platform-components */
 import React from 'react';
 import { BackAndroid, NavigationExperimental, Platform } from 'react-native';
 
+import { Action } from '../types';
```

Comment:
Yes, I will write down everything we need to do with current "flow implementation"